<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionDialog</class>
 <widget class="QDialog" name="ActionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,9,1,0">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="actorLabel">
       <property name="text">
        <string>Actor : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="actorComboBox">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page_Change_Animation">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Animation : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="animCombo_d0">
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Direction : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="animDirCombo_d0">
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Forward</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Backward</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Stopped</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>No Change</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_Change_Anim_Dir"/>
     <widget class="QWidget" name="page_Change_Cursor"/>
     <widget class="QWidget" name="page_Change_Parent"/>
     <widget class="QWidget" name="page_Change_Path"/>
     <widget class="QWidget" name="page_Change_Transparency"/>
     <widget class="QWidget" name="page_Change_Z_Depth"/>
     <widget class="QWidget" name="page_Collision_State"/>
     <widget class="QWidget" name="page_Conditional_Action"/>
     <widget class="QWidget" name="page_Create_Actor"/>
     <widget class="QWidget" name="page_Create_Timer"/>
     <widget class="QWidget" name="page_Destroy_Actor"/>
     <widget class="QWidget" name="page_Destroy_Timer"/>
     <widget class="QWidget" name="page_Event_Disable"/>
     <widget class="QWidget" name="page_Event_Enable"/>
     <widget class="QWidget" name="page_Follow_Mouse"/>
     <widget class="QWidget" name="page_Move_To"/>
     <widget class="QWidget" name="page_Physical_Response"/>
     <widget class="QWidget" name="page_Play_Music"/>
     <widget class="QWidget" name="page_Play_Sound"/>
     <widget class="QWidget" name="page_Script_Editor"/>
     <widget class="QWidget" name="page_Set_Text"/>
     <widget class="QWidget" name="page_To_Anterior_Position"/>
     <widget class="QWidget" name="page_Visibility_State"/>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="waitForFrameGroupBox">
     <property name="title">
      <string>Wait for frame action</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Animation :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="waitAnimationComboBox">
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Frames : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="waitFramesLineEdit">
          <property name="placeholderText">
           <string>0 1 2 4 7...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Note : the action will happen when the animation arrives in one of the frames above.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Frame sequence must be seperated by a space.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="executeLastActionInQueueCB">
        <property name="text">
         <string>Execute only the last action in the queue.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addActionBtn">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
